docs/Makefile

```diff
@@ -20,3 +20,27 @@ docs-up-docker:
 		-p 4000:4000 \
 		-v $(PWD):/srv/jekyll \
 		jekyll/jekyll:3.8 jekyll serve --livereload
+
+# This commands is helpful when you want to convert image to the webp format
```

Comment:
docs in the root folder, such as https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md, could have references to images in `docs` folder.

Reply:
Updated README.md